changePassword.html

```diff
@@ -1,7 +1,32 @@
 <html>
```

Comment:
Missing <!DOCTYPE html> for refference you can check https://www.w3schools.com/tags/tag_doctype.asp